hack/config.toml

```diff
@@ -0,0 +1,8 @@
+log-level = "info"
+cert-file = "/etc/pki/consumer/cert.pem"
+key-file = "/etc/pki/consumer/key.pem"
+ca-root = ["/etc/pki/consumer/ca.pem"]
+path-prefix = "api/flotta-management/v1"
+protocol = "http"
```

Comment:
I know it is not for this PR but should we change the protocol to https?

Reply:
It is, in fact, https - `http` is just a confiig keyword for yggdrasil (`mqtt` vs. `http`).